static/canjs.js

```diff
@@ -1,3 +1,4 @@
+var flexibility = require("flexibility");
```

Comment:
@imaustink, could you update this PR to conditionally load this module for only IE 9 & 10? I think [steal-conditional’s boolean conditionals](https://stealjs.com/docs/steal-conditional.html#boolean-conditionals) would be a good fit for this, but I could be wrong.